metrics/xnli/README.md

```diff
@@ -0,0 +1,99 @@
+# Metric Card for XNLI
+
+## Metric description
+
+The XNLI metric allows to evaluate a model's score on the [XNLI dataset](https://huggingface.co/datasets/xnli), which is a subset of a few thousand examples from the [MNLI dataset](https://huggingface.co/datasets/glue/viewer/mnli) that have been translated into a 14 different languages, some of which are relatively low resource such as Swahili and Urdu.
+
+As with MNLI, the task is to predict textual entailment (does sentence A imply/contradict/neither sentence B) and is a classification task (given two sentences, predict one of three labels).
+
+## How to use 
+
+The XNLI metric is computed based on the `predictions` (a list of predicted labels) and the `references` (a list of ground truth labels).
+
+```python
+from datasets import load_metric
+xnli_metric = load_metric("xnli")
+predictions = [0, 1]
+references = [0, 1]
+results = xnli_metric.compute(predictions=predictions, references=references)
+```
+
+## Output values
+
+The output of the XNLI metric is simply the `accuracy`, i.e. the proportion of correct predictions among the total number of cases processed, with a range between 0 and 1 (see [accuracy](https://huggingface.co/metrics/accuracy) for more information). 
+
+### Values from popular papers
+The [original XNLI paper](https://arxiv.org/pdf/1809.05053.pdf) reported accuracies ranging from 59.3 (for `ur`) to 73.7 (for `en`) for the BiLSTM-max model.
+
+For more recent model performance, see the [dataset leaderboard](https://paperswithcode.com/dataset/xnli).
+
+## Examples 
+
+Maximal values:
+
+```python
+from datasets import load_metric
+xnli_metric = load_metric("xnli")
+predictions = [0, 1]
+references = [0, 1]
+results = xnli_metric.compute(predictions=predictions, references=references)
+print(results)
```

Comment:
from the discussion in https://github.com/huggingface/datasets/pull/3947 we can use doctest-style code snippets to distinguish between code lines and outputs

```suggestion
>>> from datasets import load_metric
>>> xnli_metric = load_metric("xnli")
>>> predictions = [0, 1]
>>> references = [0, 1]
>>> results = xnli_metric.compute(predictions=predictions, references=references)
>>> print(results)
```